httpx_retries/retry.py

```diff
@@ -199,13 +217,13 @@ def _calculate_sleep(self, headers: Union[httpx.Headers, Mapping[str, str]]) ->
         # Fall back to backoff strategy
         return self.backoff_strategy() if self.attempts_made > 0 else 0.0
 
-    def sleep(self, response: httpx.Response) -> None:
+    def sleep(self, response: httpx.Response | httpx.HTTPError) -> None:
```

Comment:
this syntax [isn't compatible with python 3.9](https://github.com/will-ockmore/httpx-retries/actions/runs/13474874596/job/37765878408?pr=6), which we support